openpype/pipeline/colorspace.py

```diff
@@ -438,13 +438,14 @@ def get_imageio_file_rules(project_name, host_name, project_settings=None):
 
     # get file rules from global and host_name
     frules_global = imageio_global["file_rules"]
-    frules_host = imageio_host["file_rules"]
+    # host is optional, some might not have any settings
+    frules_host = imageio_host.get("file_rules", {})
 
     # compile file rules dictionary
     file_rules = {}
     if frules_global["enabled"]:
         file_rules.update(frules_global["rules"])
-    if frules_host["enabled"]:
+    if frules_host.get("enabled"):
```

Comment:
How about that IF a host has imagio setting that it must have the enabled toggle? Or is there no default structure/schema we apply in settings to all hosts if it were to exist?
```suggestion
    if frules_host and frules_host["enabled"]:
```
This would at least error and tell us the `enabled` key doesn't exist. The current code would've instead just silently skipped the file rules.

Reply:
Well, yes, this would also work. Yes, it uses default host schema and every file rule or config is having `enabled` toggle.